pkg/injector/annotations/annotations.go

```diff
@@ -17,51 +17,53 @@ package annotations
+	KeyAPITokenSecret                   = "dapr.io/api-token-secret" /* #nosec */
+	KeyAppTokenSecret                   = "dapr.io/app-token-secret" /* #nosec */
+	KeyLogAsJSON                        = "dapr.io/log-as-json"
+	KeyAppMaxConcurrency                = "dapr.io/app-max-concurrency"
+	KeyEnableMetrics                    = "dapr.io/enable-metrics"
+	KeyMetricsPort                      = "dapr.io/metrics-port"
+	KeyEnableDebug                      = "dapr.io/enable-debug"
+	KeyDebugPort                        = "dapr.io/debug-port"
+	KeyEnv                              = "dapr.io/env"
+	KeyCPULimit                         = "dapr.io/sidecar-cpu-limit"
+	KeyMemoryLimit                      = "dapr.io/sidecar-memory-limit"
+	KeyCPURequest                       = "dapr.io/sidecar-cpu-request"
+	KeyMemoryRequest                    = "dapr.io/sidecar-memory-request"
+	KeySidecarListenAddresses           = "dapr.io/sidecar-listen-addresses"
+	KeyLivenessProbeDelaySeconds        = "dapr.io/sidecar-liveness-probe-delay-seconds"
+	KeyLivenessProbeTimeoutSeconds      = "dapr.io/sidecar-liveness-probe-timeout-seconds"
+	KeyLivenessProbePeriodSeconds       = "dapr.io/sidecar-liveness-probe-period-seconds"
+	KeyLivenessProbeThreshold           = "dapr.io/sidecar-liveness-probe-threshold"
+	KeyReadinessProbeDelaySeconds       = "dapr.io/sidecar-readiness-probe-delay-seconds"
+	KeyReadinessProbeTimeoutSeconds     = "dapr.io/sidecar-readiness-probe-timeout-seconds"
+	KeyReadinessProbePeriodSeconds      = "dapr.io/sidecar-readiness-probe-period-seconds"
+	KeyReadinessProbeThreshold          = "dapr.io/sidecar-readiness-probe-threshold"
+	KeySidecarImage                     = "dapr.io/sidecar-image"
+	KeyAppSSL                           = "dapr.io/app-ssl"
+	KeyHTTPMaxRequestBodySize           = "dapr.io/http-max-request-size"
+	KeyHTTPReadBufferSize               = "dapr.io/http-read-buffer-size"
+	KeyGracefulShutdownSeconds          = "dapr.io/graceful-shutdown-seconds"
+	KeyEnableAPILogging                 = "dapr.io/enable-api-logging"
+	KeyUnixDomainSocketPath             = "dapr.io/unix-domain-socket-path"
+	KeyVolumeMountsReadOnly             = "dapr.io/volume-mounts"
+	KeyVolumeMountsReadWrite            = "dapr.io/volume-mounts-rw"
+	KeyDisableBuiltinK8sSecretStore     = "dapr.io/disable-builtin-k8s-secret-store" //nolint:gosec
+	KeyEnableAppHealthCheck             = "dapr.io/enable-app-health-check"
+	KeyAppHealthCheckPath               = "dapr.io/app-health-check-path"
+	KeyAppHealthProbeInterval           = "dapr.io/app-health-probe-interval"
+	KeyAppHealthProbeTimeout            = "dapr.io/app-health-probe-timeout"
+	KeyAppHealthThreshold               = "dapr.io/app-health-threshold"
+	KeyPlacementHostAddresses           = "dapr.io/placement-host-address"
+	KeyPluggableComponents              = "dapr.io/pluggable-components"
+	KeyPluggableComponentsSocketsFolder = "dapr.io/pluggable-components-sockets-folder"
```

Comment:
"sockets-folder" or "socket-folder"? 🤔 (Not sure either)

Reply:
Sockets, it should be more than one (from Dapr perspective)